Gemfile

```diff
@@ -3,10 +3,14 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '5.0.0.1'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3', group: :development
-gem 'pg', '0.18.1', group: :production
-gem 'rails_12factor', group: :production
+# # Use sqlite3 as the database for Active Record
```

Comment:
We can safely delete (instead of commenting them out) these lines because we use version control.